core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotationUtil.java

```diff
@@ -109,6 +124,29 @@ public static <T extends Annotation> T findFirst(AnnotatedElement element, Class
         return found.isEmpty() ? null : found.get(0);
     }
 
+    /**
+     * Returns an annotation of the {@link Description} if it is found from one of the following:
+     * <ul>
+     *     <li>the specified {@code element}</li>
+     *     <li>the super classes of the specified {@code element} if the {@code element} is a class or a method
+     *     or a parameter</li>
+     *     <li>the meta-annotations of the annotations specified on the {@code element}
+     *     or its super classes</li>
+     * </ul>
+     * Otherwise, {@code null} will be returned.
+     *
+     * @param element the {@link AnnotatedElement} to find the first annotation
+     */
+    @Nullable
+    public static Description findFirstDescription(AnnotatedElement element) {
+        final EnumSet<FindOption> options = EnumSet.of(FindOption.LOOKUP_SUPER_CLASSES,
+                                                       FindOption.LOOKUP_META_ANNOTATIONS,
+                                                       FindOption.LOOKUP_SUPER_METHODS,
+                                                       FindOption.LOOKUP_SUPER_PARAMETERS);
```

Comment:
Thanks! By the way, is it better to declare a static variable with `ImmutableSet` class to explicitly show the object is immutable?

Reply:
That's a good question. I don't mind. It seems like we just use `Set` in the declaration part because the field name is all uppercased anyway, implying it's a constant.

Reply:
Thanks! Let me use `Set` with the uppercase in variable name in order for reader to figure out that is immutable just with variable name.